frontend/src/app/core/services/google-analytics.service.ts

```diff
@@ -5,29 +5,33 @@ const GOOGLE_ANALYTICS_ID = 'UA-80363801-4';
   providedIn: 'root'
 })
 export class GoogleAnalyticsService {
+  gtag: any;
   constructor() {
+    if (typeof gtag !== 'undefined') {
```

Comment:
I didn't do it before because I am not sure whether `gtag` is ready  when `GoogleAnalyticsService` is initialized,